app/scripts/views/confirm.js

```diff
@@ -41,14 +44,20 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
     },
 
     context: function () {
-      return {
+      var context = {
         email: this.getAccount().get('email')
       };
+
+      context.experiments = this._experiments.getContext();
+      this._experiments.signal('openGmail', 'triggered');
```

Comment:
I dig it. IIUC, all experiment related context is not added directly to the context, but rather under the `experiments` property of the context. So inside of a template, you'd do something roughly like:

```
{{#experiments.gmailAvailable}}
  <a href="https://gmail.com">Open Gmail</button>
{{/experiments.gmailAvailable}}
```